### DISCUSSION TITLE
Cross compile instruction

### DISCUSSION THREAD

**Discussion by david-bouthillier** [alsa-project/alsa-utils#265] [Q&A]
I'm attempting to cross compile for Linux ARM 64 bit but I continue to encounter errors.

Here is the output of running `gitcompile`
```
configure.ac:10: error: possibly undefined macro: AM_GNU_GETTEXT
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:11: error: possibly undefined macro: AM_GNU_GETTEXT_VERSION
configure.ac:24: error: possibly undefined macro: AM_PATH_ALSA
./configure: line 3293: syntax error near unexpected token `external'
./configure: line 3293: `AM_GNU_GETTEXT(external)'
```

I also tried to compile using these instructions. https://github.com/alsa-project/alsa-utils/issues/182

However, that also results in errors.

```
configure.ac:24: error: possibly undefined macro: AM_PATH_ALSA
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
autoreconf: error: /opt/tdx-xwayland/6.4.0/sysroots/x86_64-tdxsdk-linux/usr/bin/autoconf failed with exit status: 1
```

Does anyone have build instructions for cross compiling?

**Comment by symdeb** [comment]
Followed the same steps to compile for ARM 32 bit 
	aclocal
	gettextize
	autoheader
	automake --foreign --copy --add-missing
	autoconf
	./configure
	make

and got:

/configure: line 3293: syntax error near unexpected token `external'
./configure: line 3293: `AM_GNU_GETTEXT(external)'
make: *** [Makefile:433: config.status] Error 2

with 

aclocal --force -I m4
autoreconf -fiv
gettextize
autoheader
automake --foreign --copy --add-missing
./configure --disable-alsatest --disable-alsaconf --disable-bat --disable-xmlto --with-curses=ncursesw
make

got

Checking pkg-config is at least version 0.9.0... yes
./configure: line 16245: syntax error near unexpected token `1.2.13'
./configure: line 16245: `AM_PATH_ALSA(1.2.13)'
make: *** [Makefile:433: config.status] Error 2
